pkg/provider/provider_test.go

```diff
@@ -302,31 +343,47 @@ func TestIsWorkerNode(t *testing.T) {
 		},
 		{
 			node: &corev1.Node{
-				ObjectMeta: metav1.ObjectMeta{Labels: map[string]string{"node-role.kubernetes.io/master": ""}},
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"node-role.kubernetes.io/master": ""},
```

Comment:
Should this be `control-plane` instead of `master`?  The string has been deprecated.

Reply:
This is change is just re-formatting of the current code with golines. Maybe we can address this in another PR ?

Reply:
Yes of course!